CONTRIBUTING.md

```diff
@@ -0,0 +1,15 @@
+# Contributing
+In order to contribute,
+ - Fork this repo
+ - Create a Branch on your forked repo
+ - Developer features and Create a Git Commit
```

Comment:
Line 5 is not very clear. I think maybe it should say: 'Add your changes and then create a Git commit'